Убираю проверку включен ли фильтр стоп-слов из контроллера в service

MaksimovMA opened the pull request.

MaksimovMA committed:
Убираю проверку включен ли фильтр стоп-слов из контроллера в service

ghost commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Комментарии и фильтр bad words](https://trello.com/c/OKPmW94C/276-%D0%BA%D0%BE%D0%BC%D0%BC%D0%B5%D0%BD%D1%82%D0%B0%D1%80%D0%B8%D0%B8-%D0%B8-%D1%84%D0%B8%D0%BB%D1%8C%D1%82%D1%80-bad-words)

tediik commented on src/main/java/com/jm/online_store/service/impl/BadWordsServiceImpl.java:
```diff
@@ -96,22 +98,24 @@ public void importWord(String words) {
-            String findMe = wordCheck.getBadword().toLowerCase();
-            if (textForCheck.matches(".*?\\b" + findMe + "\\b.*?")) {
-                foundWords.add(findMe);
+        if (checkEnabledCheckText()) {
+            String textForCheck = checkText.toLowerCase();
+            List<BadWords> allActiveBW = findAllWordsActive();
+            for (BadWords wordCheck : allActiveBW) {
+                String findMe = wordCheck.getBadword().toLowerCase();
+                if (textForCheck.matches(".*?\\b" + findMe + "\\b.*?")) {
+                    foundWords.add(findMe);
+                }
```
а теперь еще проверь, что тесты на этот класс написанные работают, пожалуйста

MaksimovMA committed:
Merge branch 'dev' into refactor/badWordsEnabled

MaksimovMA committed:
Поправляю тест метода "void CheckComment()"
Устанавливаю в CommonSettings для "bad_words_enabled" параметр textValue("true")(Для правильного отображения переключателя для фильтра стоп-слов в кабинете администратора)
